### PR TITLE
describe how to use forked parse-server in custom docker image

### PR DESCRIPTION
**Pull request by mman**
### New Pull Request Checklist

- [x ] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).

### Issue Description
I am trying to improve the wording around how to run a custom forked version of parse-server via docker, so that feature branches and bug fixes can be tested in the staging/dev environments before creating a pull request.

- [x ] Add changes to documentation (guides, repository pages, in-code descriptions)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ Once you have babel running in watch mode, you can start making changes to parse
 * The `lib/` folder is not committed, so never make changes in there.
 * Always make changes to files in the `src/` folder.
 * All the tests should point to sources in the `lib/` folder.
+* The `lib/` folder is produced by `babel` using either the `npm run build`, `npm run watch`, or the `npm run prepare` step.
+* The `npm run prepare` step is automatically invoked when your package depends on forked parse-server installed via git for example using `npm install --save git+https://github.com/MY_USERNAME/parse-server#MY_FEATURE`.
```
Comment (mtrezza) [inline CONTRIBUTING.md:84]: To not reference a potentially real GitHub user here, please change to:

> * The `npm run prepare` step is automatically invoked when your package depends on forked parse-server installed via git for example using `npm install --save git+https://github.com/[username]/parse-server#[branch/commit]`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -92,6 +94,9 @@ Once you have babel running in watch mode, you can start making changes to parse
 *Question*: How do I deploy my forked version on my servers?<br/>
 *Answer*: In your `package.json`, update the `parse-server` dependency to `https://github.com/MY_USERNAME/parse-server#MY_FEATURE`. Run `npm install`, commit the changes and deploy to your servers.
 
+*Question*: How do I deploy my forked version using docker?<br/>
+*Answer*: In your `package.json`, update the `parse-server` dependency to `https://github.com/MY_USERNAME/parse-server#MY_FEATURE`. Make sure the `npm install` step in your `Dockerfile` is running under non-privileged user for the ``npm run prepare`` step to work correctly. For official node images from hub.docker.com that non-privileged user is `node` with `/home/node` working directory.
```
Comment (mtrezza) [inline CONTRIBUTING.md:98]: To not reference a potentially real GitHub user here, please change to:

`https://github.com/[username]/parse-server#[branch/commit]`

**Review by mtrezza — APPROVED** [review]
Thanks for adding this! Just a nit..

**Review by mtrezza — CHANGES_REQUESTED** [review]
I meant to request the changes I mentioned earlier.

**Review by mtrezza — APPROVED** [review]
Thanks for his PR!

**Review by davimacedo — APPROVED** [review]

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0](https://github.com/parse-community/parse-server/releases/tag/5.0.0)

**Pull request merged**